src/osd/osd_types.h

```diff
@@ -4370,6 +4370,7 @@ struct store_statfs_t
   int64_t compressed = 0;              // Bytes stored after compression
   int64_t compressed_allocated = 0;    // Bytes allocated for compressed data
   int64_t compressed_original = 0;     // Bytes that were successfully compressed
+  int64_t garbage_collected = 0;       // Bytes written to recover space trapped because of overlapped blobs.
```

Comment:
Currently we measure amount of data rewritten during GC. Perhaps more useful parameter is space saved by GC procedure, i.e. the difference between previous amount of wasted bytes (allocated - useful) and the resulting one after GC.